pydecred/rpc.py

```diff
@@ -60,21 +61,21 @@ def getBlockchainInfo(self):
         """
         return GetBlockChainInfoResult.parse(self.call("getblockchaininfo"))
 
-    def getRawTransaction(self, txid, verbose=0):
+    def getRawTransaction(self, txid, verbose=False):
         """
         Returns information about a transaction given its hash.
 
         Args:
             txid (str): The hash of the transaction
-            verbose (int): Optional. Default=0. Specifies the transaction is
-                returned as a JSON object instead of a hex-encoded string
+            verbose (bool): Optional. Default=False. Specifies the transaction is
+                returned as a msgtx.MsgTx object instead of a hex-encoded string
 
         Returns:
-            RawTransactionResult or string: RawTransactionResult if verbose=1,
-                hex string for the transaction if default
+            msgtx.MsgTx or string: msgtx.MsgTx if verbose, hex string for the
+                transaction if default.
         """
-        res = self.call("getrawtransaction", txid, verbose,)
-        return RawTransactionsResult.parse(res) if verbose else res
+        res = self.call("getrawtransaction", txid, 0)
+        return MsgTx.deserialize(ByteArray(res)) if verbose else res
```

Comment:
Sorry if I mislead you here. I'm looking for `RawTransactionResult` for `verbose=true`, otherwise `MsgTx`, which can be deserialized from the hex string.

Reply:
so msgTx for verbose=False and RawTransactionResult for True